packages/react-devtools-shared/src/devtools/views/SearchInput.js

```diff
@@ -68,33 +72,33 @@ export default function SearchInput(props: Props) {
     // Here we use portals to render individual tabs (e.g. Profiler),
     // and the root document might belong to a different window.
     const ownerDocument = inputRef.current.ownerDocument;
-    ownerDocument.addEventListener('keydown', handleWindowKey);
+    ownerDocument.addEventListener('keydown', handleKeyDown);
 
-    return () => ownerDocument.removeEventListener('keydown', handleWindowKey);
+    return () => ownerDocument.removeEventListener('keydown', handleKeyDown);
   }, [inputRef]);
```

Comment:
are refs supposed to be in effect dependencies? i thought there might be a lint rule against that haha

in any case, should it be inputRef.current, or not have this dep at all?

Reply:
Ah, right. Refs aren't reactive. (Changing a ref won't trigger a re-render of a component, so it may not trigger your effect.) So yeah, not necessary in this case.

Reply:
Yup. This is an unnecessary dep.